manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
+PackageLocale: en-US
+Publisher: LibRetro
+PackageName: RetroArch

```

Comment:
```suggestion
PackageName: RetroArch Nightly
```

Reply:
putting this here so people on production version doesn't get automatically upgraded to the nightly version